### PR TITLE
Update README.md for pdftotext

### PR DESCRIPTION
**Pull request by mrvisser**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,6 +116,16 @@ sudo apt-get install pdf2htmlEX
...
+##### pdftotext (only if preview processor is desired)
+
+The version of `pdftotext` required comes with a library called Poppler which is installed when you install `pdf2htmlEX`. Note that this version of `pdftotext` is around **0.29**. The latest stand-alone version of `pdftotext` is around 3+ and it is not supported.
+
+  * If you installed `pdf2htmlEX` with the Ubuntu PPA, then you probably already have the correct version of pdftotext in your path
+  * If you installed `pdf2htmlEX` with Homebrew for MacOSX, then you probably have it available. If you can't find it on your path, try running `brew unlink poppler` then `brew link poppler`, and that should ensure all binary links (including `pdftotext`) get installed to your path
+  * If you compiled `pdf2htmlEX` yourself, you're a wizard. `pdftotext` is available in your poppler distribution's `bin` directory
+
+If none of these work for you, you can download and compile the latest [Poppler](http://poppler.freedesktop.org/releases.html) tarball. `pdftotext` will be available in the installed `bin` directory.
+
```
Comment (nicolaasmatthijs) [inline README.md]: Can you add a note about `pdftotext is used to ...`

**Comment by nicolaasmatthijs** [comment]
Left some feedback. Back to @mrvisser

File: README.md (hunk 1 above)
Comment (mrvisser) [inline README.md]: Fixed

**Comment by mrvisser** [comment]
Back to @nicolaasmatthijs

**Comment by nicolaasmatthijs** [comment]
Looks good. Merged.

**Pull request merged**